OmiseSDK/Sources/Views/Screens/Choose Payment Methods/Other Payments/EContext/EContextPaymentFormProtocol.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+protocol EContextPaymentFormViewModelProtocol {
+    var input: EContextPaymentFormViewModelInput { get }
+    var output: EContextPaymentFormViewModelOutput { get }
+}
+
+protocol EContextPaymentFormViewModelInput {
+    func startPayment(name: String, email: String, phone: String)
+    func set(loadingHandler: Handler<Bool>)
+}
+
+protocol EContextPaymentFormViewModelOutput {
+    var nameError: String { get }
+    var emailError: String { get }
+    var phoneeError: String { get }
```

Comment:
it's just a typo, right? 🤔 
```suggestion
    var phoneError: String { get }
```